src/_scss/pages/modals/aboutTheData/_aboutTheData.scss

```diff
@@ -64,6 +64,11 @@
           tr {
             td {
               border: none;
+              .usda-loading-animation__container {
```

Comment:
I'll delete this in the clean up ticket. The component library update should handle this.